engine/apps/api/views/user.py

```diff
@@ -391,6 +394,20 @@ def make_test_call(self, request, pk):
 
         return Response(status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=["post"], throttle_classes=[TestCallThrottler])
+    def send_test_push(self, request, pk):
```

Comment:
Consider adding a unit test for this endpoint, something like [this](https://github.com/grafana/oncall/blob/62412076a2f83ccf7ab028938d81d17b6f5b9d02/engine/apps/mobile_app/tests/test_user_settings.py#L7-L34)